libpod/oci_conmon_linux.go

```diff
@@ -441,7 +441,7 @@ func (r *ConmonOCIRuntime) StopContainer(ctr *Container, timeout uint, all bool)
 		}
 
 		if err := waitContainerStop(ctr, time.Duration(timeout)*time.Second); err != nil {
-			logrus.Infof("Timed out stopping container %s, resorting to SIGKILL: %v", ctr.ID(), err)
+			logrus.Warnf("Timed out stopping container %s, resorting to SIGKILL: %v", ctr.ID(), err)
```

Comment:
We should not be printing the error in the WARN-level logs. Make that a debug-level IMO.